charts/orchestrator/templates/serviceaccount.yaml

```diff
@@ -1,4 +1,24 @@
 {{- if .Values.serviceAccount.create -}}
+#******************************************************************************
```

Comment:
License should be at the top of the file.

Reply:
If it will be at top it will not get deployed. It throws error.
I have already tried and tested in all the repo